### PR TITLE
Added content in Readme.md file

### PR DESCRIPTION
**Pull request by pragapraga**
@apoorva-mk @lkcbharath @isunitha98selvan Improved Readme as far i have analysed, please do review and provide feedback 😄

### CODE REVIEW

**Comment by pragapraga** [comment]
Raising for #27

**Comment by pragapraga** [comment]
@isunitha98selvan I have added contributions.md https://github.com/acm-w-nitk/acm-w-nitk.github.io/pull/32/commits/3f1f416897dbcd958350b137409a5fef9a56f7fa . please take a look

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+```
+git clone <your forked repo URL>
+```
+- Open index.html
+- Enhance the stuff requested.
+
+```
+<br>
+## Hacktoberfest 
+PR's are welcome. Lets do it in good spirit. Claim the issue you want to work on.
```
Comment (apoorva-mk) [inline contribution.md]: Hi, a more relevant file has been added instead. So please add a link to that file in the readme for new contributor :)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+<img width="180" alt="Screenshot 2019-10-06 at 12 43 03 AM" src="https://user-images.githubusercontent.com/6701828/66259727-5105e500-e7d2-11e9-9bc8-dffa5b920710.png"><br>
+Hey There! :smile:
+ Welcome to **A**ssociation for **W**omen in **C**omputing **M**achinery. ACM-W is an international organisation which supports and empowers women in all aspects of the computing field. We at ACM-W, NITK chapter wish to inspire and nurture young minds to step up and make a mark in the computing universe.
+:two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands::two_women_holding_hands:
+</br>
+Say Hello!  :wave:
+At ACM-W we're all ears. Feel free to drop in your feedback or suggestions, we'd love to hear from you. Leave us a mail or connect with us on our social media accounts.
+<br>
+Reach us via :email: at [acmw.nitk@gmail.com](mailto:%20acmw.nitk@gmail.com)
+
```
Comment (apoorva-mk) [inline README.md]: Hi, there are a few more changes we would like in this :)
- [ ] Please remove emojis other than the email one. Also, to keep it more professional remove the 'Hey there'
- [ ] Ensure it is spelt as ACM-W NITK everywhere.
- [ ] The image should be the one with transparent background (images folder)
- [ ] Remove links to club officers accounts
- [ ] Add links to our other social media accounts
- [ ] The entire expansion of ACM-W should be in bold.
- [ ] Add the link to contributing guidelines.

**Comment by apoorva-mk** [comment]
@pragapraga if you cannot work on this, we will assign it to someone else immediately. Please let us know asap :)

**Pull request closed without merging**